open-api.yaml

```diff
@@ -405,6 +405,42 @@ components:
         - exactness
         - 'release_date:asc'
       examples: []
+    typoTolerance:
+      type: object
+      description: Customize typo tolerance feature.
+      properties:
+        enabled:
+          description: Enable the typo tolerance feature.
+          type: boolean
+          default: true
+          nullable: false
+        disableOnAttributes:
+          description: Disable the typo tolerance feature on the specified attributes.
+          type: array
+          items:
+            type: string
+          default: []
+          nullable: false
+        disableOnWords:
+          description: Disable the typo tolerance feature for a set of query terms given for a search query.
+          type: array
+          items:
+            type: string
+          default: []
+          nullable: false
+        minWordSizeForTypos:
```

Comment:
@gmourier,  

I tried the v0.27rc and I see `minWordLengthForTypo`. Wanted to confirm what name we're going with

![Screen Shot 2022-04-15 at 00 51 01](https://user-images.githubusercontent.com/90181761/163473121-708d799a-d4e7-44de-9bb9-3705e8656920.png)

Reply:
Uhm no you're right, needs to be changed

Reply:
I'll open a pr